CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # WebTorrent Desktop Version History
 
+## v0.20.0 - 2018-04-26
+
+### Added
+- Added support for additional audio extensions: 'aiff', 'ape', 'mp2', 'oga', 'opus', 'wma' (#1240)
+
+### Changed
+
+- README.md: added cask instructions (#1364)
+- Displaying filename while music metadata is being downloaded (#1361)
+- Improved the poster selection for audio/music based torrent (#1334)
+- Removed 'video-on-top' from VLC player launch options (#1286)
+- Replaced dependency `musicmetadata` with `music-metadata` (#1240)
```

Comment:
I usually skip mentioning internal changes like this that the end user will not notice